dockets/queue.py

```diff
@@ -95,18 +95,21 @@ def item_key(self, item):
                             for key_component in self.key)
         return str(item)
 
-    def move_delayed_items(self):
-        self._move_delayed_items(keys=[self._delayed_queue_key(),
-                                       self._payload_key(),
-                                       self._queue_key()],
-                                 args=[time.time(),
-                                       self.mode == self.FIFO])
+    def pop_delayed_items(self, pipeline):
+        num_popped = self._move_delayed_items(
+            keys=[self._delayed_queue_key(),
+                  self._payload_key(),
+                  self._queue_key()],
+            args=[time.time(), self.mode == self.FIFO])
```

Comment:
This was my mistake but I think technically we should remove the spaces around `==` here